.github/workflows/dependabot-batcher.yml

```diff
@@ -0,0 +1,38 @@
+# Dependabot Batcher v1.0.1
+
+name: 'Dependabot Batcher'
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "r/12.x", "develop" ]
+  pull_request:
+    branches: [ "r/12.x", "develop" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  dependabot-batcher:
+    name: 'Combine dependabot PRs'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repo'
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPENDABOT_BATCHER }}
+
+      - name: 'Set Node Version'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.8.0
+
+  # Batches Dependabot PRs into one by merging them into a combined branch, then raising a new PR
+      - name: 'Dependabot Batcher'
+        if: ${{ contains(github.head_ref, '/modules/engage-paella-player-7') }}
```

Comment:
Instead of checking this here, you can also only run on pushes or pull requests which change this patch:

https://github.com/opencast/opencast/blob/34cf94c108afa249862916829aa8b4085388b60c/.github/workflows/test-opencast-build.yml#L12-L15

Reply:
I changed to only check where paella 7 resides